JfkWebApiSkills/JfkWebApiSkills/JfkWebAPISkills.cs

```diff
@@ -18,6 +18,9 @@ namespace Microsoft.CognitiveSearch.WebApiSkills
 {
     public static class JfkWebApiSkills
     {
+        // Configure duration of SAS token for image store blobs here
+        private static readonly TimeSpan s_imageBlobSASDuration = TimeSpan.FromDays(365);
```

Comment:
Could you possibly change this to be effectively forever (i.e. some day in the year 9999) so that in case anyone uses this demo long term they don't become frustrated if it stops working and the only way to fix it is to reindex?